mifosng-android/src/main/java/com/mifos/mifosxdroid/online/savingaccounttransaction/SavingsAccountTransactionFragment.java

```diff
@@ -50,6 +51,8 @@
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
+import static android.content.Context.INPUT_METHOD_SERVICE;
+
```

Comment:
Remove this extra line. Apply the MifosStyle.xml style template to your code in Android Studio and refactor the VCS changed text using it to avoid such mistakes.